src/components/site/PageHeader.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+const PageHeader = (props) => (
+	<div className="dg_page-header">
+		<div className="dg_page-header__box ">
+			<div className="col-12 col-lg-5 col-xl-6">
+				<span className="dg_page-header__pre-title">{props.preTitle !== undefined ? props.preTitle : null}</span>
+				<span className="dg_page-header__title">{props.title} </span>
+			</div>
+			<div className="col-12 col-lg-7 col-xl-6">
+				<p>{props.deck !== undefined ? props.deck : null}</p>
+			</div>
+		</div>
+	</div>
+);
+
+PageHeader.propTypes = {
+	/** PreTitle to be displayed */
+	preTitle: PropTypes.string,
+	/** Title to be displayed */
+	title: PropTypes.string.isRequired,
+	/** Deck to be displayed */
```

Comment:
I believe the comment for desk should read something like. 

```js
/** A brief summary the describes the page the header is included on. */
```

Reply:
done

Reply:
Please use these, as @danfox01 is much better at words than me

preTitle: Type of agency, e.g., "Department of" or "Office of"

title: Agency name, e.g., "Budget and Finance"

deck: Brief description of the agency's function.

Reply:
done

Reply:
preTitle: Supplemental heading, e.g., "Department of"

title: Main page heading, e.g., "Budget and Finance"

deck: Brief description of the landing page or app.

this is the last one, i promise.

Reply:
done